Tests/SwiftDocCTests/Model/DocumentationMarkupTests.swift

```diff
@@ -158,6 +158,18 @@ class DocumentationMarkupTests: XCTestCase {
             XCTAssertEqual(expected, model.abstractSection?.content.map({ $0.detachedFromParent.debugDescription() }).joined(separator: "\n"))
         }
 
+        // Contains a HTMLBlock before the abstract
+        do {
+            let source = """
+            # Title
+            <!--Line a-->
+            Line b
+            ## Hello, world!
+            Discussion content.
+            """
+            let model = DocumentationMarkup(markup: Document(parsing: source, options: .parseBlockDirectives))
+            XCTAssertNotNil(model.abstractSection)
```

Comment:
Can we test that the abstract is indeed "Line b" as well? Just to make sure we're not parsing the HTML as the abstract.

Reply:
Nice catch, done